cmd/manager/main.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"runtime"
+
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+	"github.com/operator-framework/operator-sdk/pkg/leader"
+	"github.com/operator-framework/operator-sdk/pkg/ready"
+	sdkVersion "github.com/operator-framework/operator-sdk/version"
+	"github.com/zvonkok/special-resource-operator/pkg/apis"
```

Comment:
This should change to nvidia/gpu-operator/pkg/apis?

Reply:
Yep.